lit-action-solana-decrypt-and-sign/nodejs/src/litAction.js

```diff
@@ -0,0 +1,41 @@
+const {
+  clusterApiUrl,
+  Connection,
+  Keypair,
+  PublicKey,
+  SystemProgram,
+  Transaction,
+} = require("@solana/web3.js");
+const bs58 = require("bs58");
+
+(async () => {
+  const solanaPrivateKey = await Lit.Actions.decryptAndCombine({
```

Comment:
can we add more error catching throughout this function and either use
- `setResponse`
- `console.log` (will get added to the response from `executeJs` in `logs`)